README.md

```diff
@@ -417,4 +415,34 @@ A timeout of 0 ms will run its callback as soon as the browser isn't busy doing
 :books: When you use `setTimeout` the callback function is placed on the callback queue and ran at the next event loop.
 There was [a nice talk about this](https://www.youtube.com/watch?v=8aGhZQkoFbQ) at JSConf EU 2014.
 
-## 13. Conditional rendering
+# Next steps
+
+That’s all – we have a functional version of React now. Lets take a closer look at what we built:
+
+-   Supports HTML elements, such as `<div />` and `<p />`
+-   Supports both functional and class components — together with proper internal state handling.
```

Comment:
"proper"... Det er vel en del av " Remove the class cache".

Reply:
Enig, fjerne dette